### ISSUE TITLE
Unsupported rom

### ISSUE THREAD

**Issue by LuchiLucs** [OneB1t/HawaiiBiosReader#11]
My Asus R7 260X stock bios is unsupported, any help?

**Comment by mirh** [comment]
@lightningterror can you provide latest binary to this poor soul?

**Comment by LuchiLucs** [comment]
I didn't notice the latest binary wasn't updated, if you can update it please since I don't have VS to compile it.

**Comment by DaniloJacques** [comment]
Check pull request by @lightningterror https://github.com/OneB1t/HawaiiBiosReader/pull/10/commits/910f977c16eafdc072f3ecf6810a9c2381e0ad8a

**Comment by mirh** [comment]
Oh, lol, he didn't close it.

**Comment by lightningterror** [comment]
@mirh ofc not , just waiting for a merge.

https://drive.google.com/open?id=0B8cQAq-Ko9-uek1fd3EySUg2dWc

**Comment by LuchiLucs** [comment]
Thanks, I downloaded it. You can close the issue.
By the way I am hacking the soft powerplay table of my asus R7 260X stock bios. I am doing some trial and error through the same table on the registy first and later I will flash the bios if I am stable. I successfully edited the power states from p0 to p7 of both gpu clock and memory (the frequencies), the power limit and the fan control. I am trying to find the voltages now.

**Comment by DaniloJacques** [comment]
@jLuca what you want to do? Improve the clock range? (it's useful for 1000/1250 cards, that are locked to 1500MHz RAM)

**Comment by LuchiLucs** [comment]
No, my gpu suffers from random freezes (forced to reboot), it's caused by Windows drivers since Linux ones or Windows default ones doesn't give me this problem. I thought it was the fast switching and the range difference between 2d and 3d mode so I forced always the 3d mode with clockblocker or other utilities and this workaround solved my probblem but now since Windows CU upgrade I get those freezes even if I run always in 3d mode. I am trying to hack all I can to try to fix it. I made a post with some details about it (not updated): https://community.amd.com/message/2765120
